front_end/panels/react_devtools/ReactDevToolsView.ts

```diff
@@ -94,38 +91,79 @@ export class ReactDevToolsViewImpl extends UI.View.SimpleView {
       send: (event, payload): void => this.sendMessage(event, payload),
     };
 
-    // To use the custom Wall we've created, we need to also create our own "Bridge" and "Store" objects.
-    this.bridge = ReactDevTools.createBridge(this.wall);
-    this.store = ReactDevTools.createStore(this.bridge);
-
     // Notify backend if Chrome DevTools was closed, marking frontend as disconnected
-    window.addEventListener('beforeunload', () => this.bridge.shutdown());
+    window.addEventListener('beforeunload', () => {
+      console.debug('[ReactDevToolsView] beforeunload');
+      this.bridge?.shutdown()
```

Comment:
Is optional chaining required here?

Reply:
Yes because `this.bridge` is no longer initialized in the constructor.